master/internal/stream/trials.go

```diff
@@ -63,7 +63,7 @@ func (tm *TrialMsg) SeqNum() int64 {
 // UpsertMsg creates a Trial stream upsert message.
```

Comment:
Don't we also want to remove `upsertCache` and `deleteCache` from the message, since it's no longer being used?

Reply:
right, good point